jenkins/image_building/dib_elements/centos-ci/post-install.d/02_configure

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Configure network (set nameservers and disable peer DNS).
+set -eux
+
+sudo sed -i "0,/.*PermitRootLogin.*/s//PermitRootLogin yes/" /etc/ssh/sshd_config
+
+# RESET CLOUD INIT
+# Following will remove any cloud init's previous run
+# data and force cloud-init to again on next boot.
+
+sudo rm -rf /var/lib/cloud/*
```

Comment:
Do we need this? DIB is not using cloud-init so I guess we can remove it.

Reply:
Thank you for noticing this, I copied this from packer image building workflow. I did not pay attention to that. I will remove now.